### DISCUSSION TITLE
Change the name of the output folder and the path of the files in it

### DISCUSSION THREAD

**Discussion by c2tz** [rendercv/rendercv#56] [Q&A] [answered]
I would like to change the name of the `rendercv_output` folder to just "output". What should I do?

Can I also change the output path of the files, for example I'd like to have only the pdf directly in the root of my repo rather than in this `rendercv_output` folder (I know what I'm asking is possible with github-actions but directly from rendercv).

**Comment by sinaatalay** [comment, marked as answer]
This is currently not possible, but I can implement these as CLI arguments. Could you please open an issue for this feature (also, an issue for pngs would be great)? Thank you!

**Comment by c2tz** [comment]
https://github.com/sinaatalay/rendercv/issues/58
